### ISSUE TITLE
Add tutorial to the documentation website

### ISSUE THREAD

**Issue by pepicrft** [tuist/tuist#2962]
## Context 🕵️‍♀️
Our current website has in-depth documentation about all the available commands and the manifest models. That's useful for people that are already using the project. However, for a person wanting to learn about Tuist for the first time, it can feel overwhelming and hard to find the starting point.

## What 🌱
We should add a tutorial to the documentation website that guides a new user through the following steps:
- Install Tuist and touch on version management through `tuistenv`.
- Initialize a new project with `tuist init` and run `tuist generate --open` to open it in Xcode.
- Edit the project with `tuist edit` and introduce them to the concept of helpers.
- Build and test the project using `tuist build` and `tuist test`.
- Introduce them to the concept of focused projects through the `tuist focus` command.

**Comment by ronanociosoig-200** [comment]
It would be nice to do it all in SwiftUI, and have some screen designs done as a project starter.

**Comment by VilleWitt** [comment]
v3.0: `tuist focus <XYZ>` mentioned has been merged with `tuist generate <XYZ>`

**Comment by VilleWitt** [comment]
TL;DR: `tuist init --template swiftui` bootstraps a SwiftUI version. 🤯🚀🥂 This should be mentioned in the tutorial.

Longer story: Im a newbie in development for UIs in the Apple world. I have only been using SwiftUI - not UIKit. I try to do as little in Xcode project settings as possible as it is a loosely coupled nightmare of inconsistency. That painpoint and git merge conflicts for simple stuff such as file ordering in xcodeproj files made me look for other was of working with Apple development. This is how I found Tuist. Trying to change the default bootstrapped version from UIKit to SwiftUI I got entangled in stuff I had no idea about. I ditched Tuist for some time and went back to the "official" painful way to get stuff done. Knowing about the SwiftUI template could have saved be hours, if not days.

In fact, knowing about the swift template could probably have saved me from ditching `tuist` and go back to the "official XCode workflow" to get stuff done. After some time the pain brought me back to `tuist`, but my first experience made me look much more at `bazel`, and others. Just being honest.

Switching from the default template to swiftui might attract more newbies, and old timers are probably less confused. 😉 Anyhow, I believe it makes sense to mention the option of `--template swiftui` in the tutorial to get people onboarded.
